### PR TITLE
Chapter on how to switch from V-REP to a real robot

### PR DESCRIPTION
**Pull request by pierre-rouanet**
I started to write a chapter on this topic (a few images are still missing though). It will particularly useful in the context where people are first developing on a simulator and at some point decide to switch to a real robot. 

At the moment I don't think we ever explain how this can be done.

@damiencaselli @show0k @oudeyer review would be much appreciated!

### CODE REVIEW

File: 07-from-simulation-to-real-robot/03-using-jupyter.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+One of the advantages of working with Jupyter Notebooks is the possibility to use it as a client/server approach. Each Poppy robot hosts a Jupyter server that can be accessed via the web interface (see [section quickstart](#use-the-web-interface) for details).
+
+When working in simulation, everything is run and stored locally on your computer. While when working with a real robot you can program it from a web browser on your own machine but your notebooks are actually stored and run in the robot.
+
+Thus to switch from simulation to a real robot, you need to switch from your local instance of Jupyter to the remote one hosted by the robot. The steps are described below.
+
+### Connect to the Jupyter on the robot
+
+Once connected to the robot web interface (we will assume here its hostname is *poppy.local*, just replace it by the new hostname if you changed it), you should see a **open Ipython notebook** link.
```
Comment (show0k) [inline 07-from-simulation-to-real-robot/03-using-jupyter.md]: the hostname is poppy not poppy.local

File: 07-from-simulation-to-real-robot/03-using-jupyter.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+The Python installed on the robot is Python 2.7 and comes with most of the scientific main libraries (numpy, scipy, matplotlib, opencv). An exhaustive list of the installed python packages will be available soon (TODO!). At the moment, the easier way to obtain it, is to used a *terminal notebook* which can be directly run from the Jupyter interface.
+
+![Launch a terminal](../img/jupyter/open-terminal.jpg)
+
+![Jupyter terminal](../img/jupyter/terminal.jpg)
+
+
+Using the same technique, you can install 3rd pary libraries directly on the robot. The [pip](https://pip.readthedocs.org) and [conda](http://conda.pydata.org/docs/) utility tool are installed and should be used when possible.
+
+*Note that the embedded board are based on armv7 and thus some libraries may be hard to compile. We are maintaining a list of conda recipies specifically built for this platform [here](https://anaconda.org/poppy-project). Contributions are more than welcomed! *
```
Comment (show0k) [inline 07-from-simulation-to-real-robot/03-using-jupyter.md:91]: Btw, it would be usefull to compile them for pyhon 3 !

**Comment by show0k** [comment]
Proofread done !

File: 07-from-simulation-to-real-robot/03-using-jupyter.md (hunk 2 above)
Comment (pierre-rouanet) [inline 07-from-simulation-to-real-robot/03-using-jupyter.md:91]: Agreed!

**Pull request merged**